gf-webapp/src/logic/rooms/roomPage/roomPromo/information/Information.tsx

```diff
@@ -17,7 +16,7 @@ export const Information = (props: InformationProps) => {
     <div>
       <div className={styles.container}>
         <div className={styles.description}>
-          {roomInfo.roomDescription[props.type]}
+          {roomInfo.descriptionShort[`${props.roomNumber}` as keyof typeof roomInfo.descriptionShort]}
```

Comment:
If someone know how to avoid construction as keyof typeof... please write me

Reply:
Create an enum with room numbers)
I'm not serious)